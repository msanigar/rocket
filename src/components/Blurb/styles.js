import styled from 'styled-components';

export const Container = styled.div`
    padding: 4em;
    background: papayawhip;
    margin: 1rem;
`;

export const Title = styled.h1`
    font-size: 1.5em;
    text-align: center;
    color: palevioletred;
`;

export const SubTitle = Title.extend`
    color: tomato;
`;