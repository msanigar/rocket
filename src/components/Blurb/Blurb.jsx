import React from 'react';
import { Container, Title, SubTitle } from './styles';

const Blurb = (props) => {
    return(
        <Container>
            <Title>This is a test component fully styled with 'Styled Component!'</Title>
        </Container>
    );
}

export default Blurb;
