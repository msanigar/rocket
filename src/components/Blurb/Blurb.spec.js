import React from 'React';
import Blurb from './Blurb';
import renderer from 'react-test-renderer';

it('Blurb should match snapshot', () => {
    const blurb = renderer
        .create(
            <Blurb />)
        .toJSON();
    expect(blurb).toMatchSnapshot();
});
