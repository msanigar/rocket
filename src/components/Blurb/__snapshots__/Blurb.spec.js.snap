// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blurb should match snapshot 1`] = `
<div
  className="sc-bdVaJa hnkImK"
>
  <h1
    className="sc-bwzfXH hsxXBG"
  >
    This is a test component fully styled with 'Styled Component!'
  </h1>
</div>
`;
